docs/design/design-rtsp-record-application.md

```diff
@@ -0,0 +1,130 @@
+# RTSP recording application 
+
+User can use this application to watch rtsp using ffmpeg or other third-party package to store the stream into PV in other host which has RTX 4090.
```

Comment:
RTX 4090 => supporting GPU that can accelerate video encoding
rtsp => RTSP, same for other occurrences